templates/fullstack/server/src/graphql.ts

```diff
@@ -1,26 +1,37 @@
 import { join } from 'path';
 import { loadFiles } from '@graphql-tools/load-files';
-import { ApolloServer, PubSub } from 'apollo-server-express';
-import { buildSchema } from 'graphql';
-import { createKnexPGCRUDRuntimeServices } from '@graphback/runtime-knex';
-import { createDB } from './db';
-import { models } from './resolvers/models';
-import resolvers from './resolvers/resolvers';
+import { ApolloServer } from 'apollo-server-express';
+import { buildGraphbackAPI } from 'graphback';
+import { createKnexDbProvider } from '@graphback/runtime-knex';
+import { migrateDB, removeNonSafeOperationsFilter } from 'graphql-migrations';
+import { createDB, getConfig } from './db';
+import { SchemaCRUDPlugin } from '@graphback/codegen-schema';
 
 /**
  * Creates Apollo server
  */
 export const createApolloServer = async () => {
   const db = await createDB();
-  const pubSub = new PubSub();
+  const dbConfig = await getConfig();
+
+  const schema = (await loadFiles(join(__dirname, '../../model/'))).join('\n');
+  const { resolvers, contextCreator, typeDefs} = buildGraphbackAPI(schema, {
+    dataProviderCreator: createKnexDbProvider(db),
+    plugins: [new SchemaCRUDPlugin({
+      outputPath: "./src/schema/schema.graphql"
+    })]
+  });
+
+  migrateDB(dbConfig, typeDefs, {
```

Comment:
Migration is performed on the template using the same configuration. 
We also support MongoDB as database and have written nice UI for both Mongo and Postgress.

Reply:
So we won't generate code anymore but we will generate the executable schema on runtime instead, right?

Reply:
Do you mean GraphQL resolvers?
There is ultimate flexibility here. Plugins giving ability to generate any code, however by default out templates do not generate resolvers and build them at runtime by default.

Point taken was that if generated resolvers are just stubs and you cannot change then (as subsequent generation will override them again) their value is very minimal.
This is a tradeoff - if we take look into runtime we have there is a lots of logic involved in filtering, pagination, mapping complex queries, connecting to existing databases and tables, ID representation, relationships etc. With graphback, you can generate your own resolvers to connect with custom logic. It brings the best from descoped projects like Prisma 1 and TypeORM and comparing to some knex wrappers like: https://www.npmjs.com/package/graphql-server-crud Graphback offers platform and swiss army knife for CRUD operations that can be integrated with other tools by utilizing https://graphqlcrud.org